pkg/pub_worker/lib/src/bin/dartdoc_wrapper.dart

```diff
@@ -179,7 +179,9 @@ Future<void> _dartdoc({
     final suffix = file.path.substring(docDir.length + 1);
     final targetFile = File(p.join(tmpOutDir, suffix));
     await targetFile.parent.create(recursive: true);
-    if (file.path.endsWith('.html')) {
+    final isDartDocPage =
+        file.path.endsWith('.html') && !file.path.endsWith('-sidebar.html');
```

Comment:
So we're not going to sanitize the sidebar?

I suspect we need a `DartDocSideBar` class.